tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware_common/algorithm.py

```diff
@@ -293,6 +293,8 @@ def _change_quantization_scope(self, model, original_accuracy_drop,
                                      for name, params in self._metrics_config.items()}
             new_accuracy_drop = metrics_accuracy_drop[metric_to_optimize]
             logger.info('Accuracy drop with the new quantization scope is %s', metrics_accuracy_drop)
+            with open('aaq_log.txt', 'a') as f_log:
```

Comment:
I think we should remove this logging. This looks like for debugging, isn't it?

Reply:
You can also change it to logger.debug(...) and flash to the default logging location.

Reply:
This is my mistake, I removed these debugging codes.